CMakeLists.txt

```diff
@@ -1,54 +1,168 @@
 cmake_minimum_required(VERSION 3.13)
 
 get_filename_component(root ${CMAKE_CURRENT_LIST_DIR} ABSOLUTE)
-include(${root}/deps/readies/cmake/main)
+get_filename_component(binroot ${CMAKE_CURRENT_BINARY_DIR}/.. ABSOLUTE)
 
+# Platform detection
+if(APPLE)
+  set(OS "macos")
+elseif(UNIX)
+  set(OS "linux")
+elseif(WIN32)
+  set(OS "windows")
```

Comment:
we can remove this

Reply:
All of the platform detection or only windows?

Reply:
windows